src/main/java/seedu/address/model/Model.java

```diff
@@ -78,14 +78,7 @@
     void commitAddressBook();
 
     /**
-     * Returns the RoomList
-     * @return
-     */
-    RoomList getRoomList();
-
-    /**
-     * Checks out the room.
-     * @param roomNumber
+     * Checks out the room by its room number and removes its registered guest.
      */
     void checkoutRoom(RoomNumber roomNumber);
```

Comment:
Should we checkout a room or a guest?

Reply:
Room only. Occupant staying in the room will be transitively checked out.